frontend/packages/pipelines-plugin/src/components/taskruns/list-page/TaskRunsListPage.tsx

```diff
@@ -16,19 +16,20 @@ const TaskRunsListPage: React.FC<Omit<
   React.ComponentProps<typeof ListPage>,
   'canCreate' | 'kind' | 'ListComponent' | 'rowFilters'
 > &
-  TaskRunsListPageProps> = ({ hideBadge, showPipelineColumn = true, ...props }) => {
+  TaskRunsListPageProps> = ({ hideBadge, showPipelineColumn = true, namespace, ...props }) => {
   const searchParams = getURLSearchParams();
   const kind = searchParams?.kind;
-  const badge = usePipelineTechPreviewBadge(props.namespace);
+  const badge = usePipelineTechPreviewBadge(namespace);
   return (
     <ListPage
-      {...props}
       customData={{ showPipelineColumn }}
       canCreate={kind?.includes(referenceForModel(TaskRunModel)) ?? false}
       kind={referenceForModel(TaskRunModel)}
       ListComponent={TaskRunsList}
       rowFilters={taskRunFilters}
       badge={hideBadge ? null : badge}
+      namespace={namespace}
+      {...props}
```

Comment:
Spread should go before, not after, static props. You are "overriding" the spread props, not hoping the spread doesn't override you. It's confusing to see a static value and it got magically overridden because the calling component decided to pass an additional prop you weren't expecting.

Undo the location change.

Reply:
Thanks for pointing it out as well as updating the title. I've rectified the changes